crm_lead_date_duration_sale/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See https://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "CRM Lead Date Duration Sale",
+    "summary": """
+        Relate Event Date field of crm.lead with Delivery Date field of sale.order.
```

Comment:
Better more human readable description:
```suggestion
        Links CRM event date management to Sales Order generation
```